setup.py

```diff
@@ -56,17 +56,17 @@ def readme():
     ],
     keywords='thumbor aws',
     install_requires=[
-        'python-dateutil',
+        'python-dateutil>=2.8',
         'thumbor>=7.0.0a2,<8',
         'aiobotocore==2.2.0',
+        'boto3>=1.9,<1.13',
```

Comment:
You'll need to update those version to match what aiobojtocore needs. I've restricted the range to reduce the build time.